src/bundle/Resources/public/js/scripts/sidebar/extra.actions.js

```diff
@@ -47,9 +47,6 @@
             backdrop.show();
             doc.body.addEventListener('click', detectClickOutside, false);
             doc.body.classList.add('ibexa-scroll-disabled');
-        } else {
-            doc.body.removeEventListener('click', detectClickOutside);
-            removeBackdrop();
```

Comment:
Will we always have backdrop if the extraActions were hidden by this event (not by clicking outside)?